packages/sdk/contractkit/src/wrappers/Governance.test.ts

```diff
@@ -347,3 +365,39 @@ testWithGanache('Governance Wrapper', (web3: Web3) => {
   //   )
   // })
 })
+
+testWithAnvil('GovernanceWrapper', (web3: Web3) => {
+  describe('Hotfixes', () => {
+    it('gets L1/L2 hotfix record for version >= 1.5.0.0', async () => {
+      const kit = newKitFromWeb3(web3)
+      const governance = await kit.contracts.getGovernance()
+      const hotfixHash = Buffer.from('0x', 'hex')
+
+      // Sanity check to make sure we're on at least 1.5.0.0 version
+      expect((await governance.version()).isAtLeast(new ContractVersion(1, 5, 0, 0)))
+
+      // Test L1 context
+      const hotfixRecordL1 = await governance.getHotfixRecord(hotfixHash)
+      expect(hotfixRecordL1).toMatchInlineSnapshot(`
+        {
+          "approved": false,
+          "executed": false,
+          "preparedEpoch": "0",
+        }
+      `)
+
+      // Test L2 context
+      await setupL2(web3)
```

Comment:
i think you are already planning on doing this but... having the L1 and L2 be separate tests cases is probably better